panel/template/fast/grid/fast_grid_template.html

```diff
@@ -54,6 +54,8 @@
   }
   #header-design-provider {
     --neutral-foreground-rest: {{ style.header_color }};
+    background: {{ style.header_background }};
```

Comment:
If we set these, we will see the header_background and header_color immediately. And not after all the `fast` web components have been initialized.